relevanceai/vector_tools/cluster.py

```diff
@@ -365,9 +365,7 @@ def cluster(
             if cluster_args is None:
                 cluster_args = CLUSTER_DEFAULT_ARGS[cluster]
             if cluster in ["kmeans", "kmedoids"]:
-                if (k is None and cluster_args is None) or (
-                    "n_clusters" not in cluster_args.keys()
-                ):
+                if k is None:
```

Comment:
Why was the `n_clusters` logic removed?

Reply:
when running the projector plot with:
`client.projector.plot(
        dataset_id=DATASET_ID,
        vector_label=vector_label,
        vector_field=vector_field,
        number_of_points_to_render=10000,
        cluster = "kmeans",
        num_clusters=20
    )`
I noticed that `num_clusters` had no effect on the number clusters displayed. I traced the issue back to the `_choose_k` function which always set `k=10` ie `num_clusters`. This happened only because I didn't specify the cluster args as well, and I shouldn't have to if I set the cluster algo and num cluster separately.

Reply:
only if `k` is not given, should the number of clusters be determined

Reply:
Yeah but the logic only holds if only `k` is considered. `n_clusters` could be an alias for `k` and should not be removed.